procedures/CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ add_standalone(wlpa)
 add_standalone(subgraph_isomorphism)
 add_standalone(sybilrank)
 add_standalone(leiden)
+add_stanalone(khop)
```

Comment:
the add_standalone command here is to compile the procedures under algo_cpp directory instead of community

Reply:
We have fixed it. Thanks!